docs/installation/SUMMA_on_OS_X.md

```diff
@@ -62,4 +65,63 @@ In the following I will assume that you don't have a Fortran compiler or NetCDF
 
  1. Now obtain the SUMMA source code from the [SUMMA source code repository](https://github.com/NCAR/summa). You may just want to download the latest tagged release. Unless you are planning to contribute to the source code, there is no need to clone or fork the repository.
 
- 1. Untar or unzip the archive, then go to the `summa/build` directory and follow the instructions in the [SUMMA installation](SUMMA_installation.md) page. If you are using MacPorts, the `FC_ENV` can be set to `gfortran-6-macports`.
+ 2. Untar or unzip the archive, then go to the `summa/build` directory and follow the instructions in the [SUMMA installation](SUMMA_installation.md) page. If you are using MacPorts, the `FC_ENV` can be set to `gfortran-6-macports`.
+
+ ## Instructions with homebrew
+
+ These instructions deviate from the standard SUMMA on OS X installation instructions in that Homebrew is used as an alternative
+ to MacPorts as a package installer. This is largely due to Catalina changes to a users /opt/ folder.
+
+ These instructions assume that you've already cloned the SUMMA repository.
+ If you haven't, follow the initial [SUMMA Installation Instructions](https://summa.readthedocs.io/en/latest/installation/SUMMA_installation/)
+
+ ### Step 1: Installation of Homebrew
+
+ If you are already using MacPorts, it is recommended to uninstall it to avoid package issues.
+ Uninstallation instructions are available [here](https://guide.macports.org/chunked/installing.macports.uninstalling.html).
+ If you'd like to save the currently installed packages for MacPorts to file, use the command: `port installed > ports_installed.txt`
+
+ Homebrew can then be installed following the instructions [here](https://brew.sh/)
+
+ ### Step 2: Installation of Required packages
+
+ #### Fortran compiler
+
+ Use the command as follows to install gcc and gfortran compilers.
+ `brew install gcc`
+
+ #### NetCDF
+
+ Use the command as follows to install netCDF.
+ `brew install netcdf`
+
+ Note that as a bonus, NetCDF Fortran is installed with this command and does not require action.
+ It is installed as a [resource](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/netcdf.rb), by default.
+
+ #### LAPACK and openblas
+
+ Use the following commands to install lapack and openblas using Homebrew
+ `brew install lapack`
+ `brew install openblas`
+
+  ### Step 3: Create the Makefile for use in Compliation
```

Comment:
Comment 1: pleas ensure consistency of step numbering between Macports and Homebrew sections.